manifest.json

```diff
@@ -10,11 +10,46 @@
         "48": "images/detective-emoji-48.png",
         "128": "images/detective-emoji-128.png"
     },
-    "background": {},
+    "browser_specific_settings": {
+        "gecko": {
+            "id": "{6b798ead-3d2d-453d-b7ef-a9733fab937a}"
+        }
+    },
     "web_accessible_resources": [
         {
             "resources": ["css/*.css"],
-            "extension_ids": ["lohigdpnlkaabdijopklelgffjopnghf"]
+            "matches": [
```

Comment:
This is here only because two keys are required here, and `extension_ids` can't be used because Chrome and Firefox expect different formatting.